DrawerKit/DrawerKit/Internal API/AnimationController.swift

```diff
@@ -28,10 +28,7 @@ extension AnimationController: UIViewControllerAnimatedTransitioning {
 
     func animateTransition(using transitionContext: UIViewControllerContextTransitioning) {
         let (presentingVC, presentedVC) = viewControllers(transitionContext, isPresentation)
-        if isPresentation {
-            let controller = presentedVC.presentationController as! PresentationController
-            controller.intermediate.view.addSubview(presentedVC.view)
-        }
+        if isPresentation { transitionContext.containerView.addSubview(presentedVC.view) }
```

Comment:
Our normal coding style is
```
if condition {
    do something
}
```
also for simple one-liners.

Reply:
Fixed in https://github.com/Babylonpartners/DrawerKit/pull/52/commits/b5e3bd79da24ca127c88c8f442ddde4282d6c73d.